web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -424,10 +429,31 @@ private void copyCustomizationIntoWebapp(ServletContext servletContext, Properti
 					}
 				}
 			}
-			
+			List<String> locales = Arrays.asList("", "_de", "_en_GB", "_es", "_fa", "_fr", "_hi", "_in_ID", "_it", "_kh",
```

Comment:
It's for production. The list contains all locales for messages which are at https://github.com/tomaszmueller/openmrs-core/tree/TRUNK-4386_fix/api/src/main/resources 

The issue here is that copying is needed if we want to preserve the functionality of being able to easily modify messages on a running server by modifying them in WEB-INF exploded from a war file. Although messages can be easily found on classpath by ReloadableResourceBundleMessageSource, modifications are only supported for messages in WEB-INF, thus we copy them from api in case someone wants to edit.

We could remove that and instead document that the user needs to copy the file himself if he wants to customize it.